### PR TITLE
Shopping cart integration

### PR DESCRIPTION
**Pull request by lvbn**
Add to shopping cart, delete from shopping cart and see all items in shopping cart seems to be working fine. There is a dependency between shopping cart products and products, and between users and shopping carts, so in order to see the features working fine, we need to create user and product first. 

I had to touch on some other files like models.ts and product-model.ts. Please have a look to see if it's everything ok. 

Also the App.ts file, I was using it in the begining, but then I saw that I was on the wrong track. Let me know if it's everything ok there as well, please...

Please let me know if there is something you think I should work on before merging. 

The shopping-cart-controller is not being used right now. But please don't delete the file. We might need it in the future. 

The shopping-cart-product-controller looks messy, there is some clean up to be done. Will do it when we get some time.

### CODE REVIEW

**Review by jamalbran — APPROVED** [review]
looks good

File: client/src/App.tsx [hunk 1]
```diff
@@ -64,7 +61,7 @@ const { loggedIn } = userStore()
               <ProductForm/>
             </>
             } />
-        
+
```
Comment (karoteblurs) [inline client/src/App.tsx:64]: Missing routes for sell and seller

**Review by sherrilla71940 — APPROVED** [review]

**Review by karoteblurs — APPROVED** [review]

File: client/src/App.tsx (hunk 1 above)
Comment (lvbn) [reply client/src/App.tsx:64]: Sell should be here now...
Seller seems to be on your code, yet to be pushed...

File: server/src/routers/shoppingcarts.ts [hunk 2]
```diff
@@ -18,10 +18,10 @@ export default function shoppingCartsRouter(router: express.Router) {
...
 
-  // Add Product to shopping cart
+  // Delete Product to shopping cart
   router.delete("/shoppingcart", deleteProductFromShoppingCart);
 
   // Get all Products from shopping cart
-  router.get("/shoppingcart", getAllProductsFromShoppingCart);
+  router.post("/shoppingcartproducts", getAllProductsFromShoppingCart);
 
 }
```
Comment (karoteblurs) [inline server/src/routers/shoppingcarts.ts:27]: FIXED METHODS POST for the product ID are not going be reflected

**Review by karoteblurs — APPROVED** [review]

**Pull request merged**